sources/packages/backend/apps/db-migrations/src/sql/CASDistributionAccounts/Create-cas-distribution-accounts.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE sims.cas_distribution_accounts(
+  id SERIAL PRIMARY KEY,
+  award_value_code VARCHAR(10) NOT NULL,
+  offering_intensity sims.offering_intensity NOT NULL,
+  operation_code CHAR(2) NOT NULL,
```

Comment:
What could be the benefit of not using enum here?

Reply:
I didn't find a strong reason to use one, so I kept it simple.
The idea was to keep the awards associated with accounts and "operation code" where operation codes are open to any string of two characters to be sent to CAS.